style.css

```diff
@@ -0,0 +1,278 @@
+	.mainContainerGrid {
+		width: 80%;
+		margin: 0 auto;
+	}
+}
+
+.colorBackgroundDark {
+	/* darker stone blue - using HSL so can easily relate color-lightness */
+	background-color:  hsl(204, 46%, 44%);
+}
+
+.colorBackgroundLight {
+	/* lighter stone blue - using HSL so can easily relate color-lightness */
+	background-color: hsl(204, 46%, 52%);
+}
+
+.sectionPaddingBottom {
+	padding-bottom: 20px;
+}
+
+.sectionPaddingTop {
+	padding-top: 20px;
+}
+
+.fontCursive {
+	 font-family:'Monotype Corsiva','Apple Chancery','ITC Zapf Chancery','URW Chancery L',cursive;
+}
+
+
+/* ------------------------------------------
+	HEADER & NAV 
+--------------------------------------------- */
+
+header {
+	text-align: center;
+	font-size: 120%;
+}
+
+.colorHeaderFooter {
+	background-color: rgb(87, 24, 103);
+	color: white;
+}
+
+.fontSizeNav {
+	font-size: 200%;
+}
+
+nav {
+	padding: 20px;
+}
+
+nav a {
+	padding: 20px;
+	color: white;
+	text-decoration: none;
+}
+
+nav a:hover {
+	background-color: rgb(61, 123, 165);
+}
+
+a {
+	color: hsl(204, 46%, 10%);
+}
+
+/* MQ for Nav */
+@media screen and (max-width: 700px) {
+	.fontSizeNav {
+		font-size: 150%;
+	}
+}
+
+
+@media screen and (max-width: 550px) {
+	nav a {
+		padding: 10px;
+	}
+	nav {
+		padding: 10px;
+	}
+}
+
+/* ---------------------------------------------------------------------
+	MAIN Section with background image & containers & content formatting 
+------------------------------------------------------------------------ */
+
+.bkgImageStyle {
+	width: 100%;
+	height: auto;	
+	background-size: cover; /* user cover to get full width, instead of contain */
+	background-repeat: no-repeat;
+	
+}
+
+.opacityBkgImages {
+	background-color: rgba(61,123,164,0.7);
+	padding-top: 10px;
+	padding-bottom: 20px;
+	
+}
+.bkgImagePicCoding {
+	background-image: url("images/bkg_dav_coding_dark_v2.jpeg");
+}
+
+.bkgImagePicAbout {
+	background-image: url("images/keyboard_chair2_v2.jpeg");
+}
+
+.bkgImagePicContact {
+	background-image: url("images/blue_book_mac2_v2.jpeg");
+}
+
+.containerFlex {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	align-content: space-around;
+	width: 100%;
+}
+
+.articleBox {
+	padding: 10px;
+	margin: 20px;
+	flex: 1 1 250px;
+	text-align: center;
+}
+
+@media screen and (min-width: 1080px) { 
+	.articleBox {
+		margin: 40px;
+	}
+}
+
+.articleBorder {
+	border: 5px solid black;
+	border-radius: 10px;
+	border-color:  hsl(204, 46%, 20%);
+}
+
+.contentIntro {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-end;
+	align-items: center;
+	width: 50%;
+	padding-left: 300px;
+}
+
+@media screen and (max-width: 550px) {
+	.contentIntro {
+		padding-left: 150px;
+	}
+}
+
+.textAboutMe {
+	color: white;
+/*	background-color: black;*/
+	background-color: rgb(87, 24, 103);
+	vertical-align: middle;
+/* need to set top & bottom margins to 0 so there is no space between the text for the Name and Role */
+	margin-top: 0px;
```

Comment:
0 values dont need px, 0 is 0 in any unit

Reply:
done